analysis/parser/RawTextParser/src/test/kotlin/de/maibornwolff/codecharta/rawtextparser/FileMetricsTest.kt

```diff
@@ -0,0 +1,40 @@
+package de.maibornwolff.codecharta.rawtextparser
+
+import de.maibornwolff.codecharta.parser.rawtextparser.FileMetrics
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+
+class FileMetricsTest {
+    @Test
+    fun `Should add metric to map when provided with valid input`() {
```

Comment:
I personally try to not mention internal implementation details in other parts of the applications like the unit test name. By that, there is a connasence of naming and you would have to rename the tests if you decide to not use a map in the future anymore.
```suggestion
    fun `Should add file metric when provided with valid metric data`() {
```